controller/config.go

```diff
@@ -52,13 +52,16 @@ const (
 
 	DefaultRaftCommandHandlerMaxQueueSize = 1000
 
+	// DefaultTlsHandshakeRateLimiterEnabled is whether the tls handshake rate limiter is enabled by default
+	DefaultTlsHandshakeRateLimiterEnabled = false
+
 	// TlsHandshakeRateLimiterMinSizeValue is the minimum size that can be configured for the tls handshake rate limiter
 	// window range
 	TlsHandshakeRateLimiterMinSizeValue = 5
 
 	// TlsHandshakeRateLimiterMaxSizeValue is the maximum size that can be configured for the tls handshake rate limiter
 	// window range
-	TlsHandshakeRateLimiterMaxSizeValue = 5000
+	TlsHandshakeRateLimiterMaxSizeValue = 10000
```

Comment:
can this be even larger and set down if people need/want to? i almost feel like it should default to max int/long/whatever

Reply:
nevermind - i guess you have to turn it on to use it anyway

Reply:
Yeah, I switched it to off by default, just to be safe.